From 0a3ad69fbece088813759c4f9baadf9162748322 Mon Sep 17 00:00:00 2001
From: Josh Wilsdon <josh@wilsdon.ca>
Date: Wed, 21 Nov 2018 12:31:57 -0800
Subject: [PATCH] TRITON-974 CNAPI should use non-broken histogram buckets
 Reviewed by: Orlando Vazquez <orlando@joyent.com> Approved by: Orlando
 Vazquez <orlando@joyent.com>

---
 package.json | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/package.json b/package.json
index a8149f7..b3196c5 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "cnapi",
   "description": "SmartDataCenter Compute Node API",
-  "version": "1.15.0",
+  "version": "1.16.0",
   "author": "Joyent (joyent.com)",
   "private": true,
   "dependencies": {
@@ -15,7 +15,7 @@
     "gc-stats": "1.2.0",
     "jsprim": "1.4.1",
     "libuuid": "0.2.1",
-    "moray": "3.5.0",
+    "moray": "git+https://github.com/joyent/node-moray.git#v3.6.0",
     "nodeunit": "0.11.0",
     "once": "1.4.0",
     "qs": "6.5.2",
@@ -24,7 +24,7 @@
     "sdc-clients": "10.0.3",
     "sprintf": "0.1.5",
     "trace-event": "1.3.0",
-    "triton-metrics": "0.3.0",
+    "triton-metrics": "1.0.0",
     "vasync": "1.6.4",
     "verror": "1.9.0",
     "wf-client": "0.2.1",
-- 
2.21.0

